designs/Prerelease-Option.md

```diff
@@ -0,0 +1,55 @@
+# Spec Name
+
+* Status: In Review
+* Author(s): [Martin Ruiz](https://github.com/martinrrm)
+* Issue: [4699](https://github.com/NuGet/Home/issues/4699) dotnet add package - support for NuGet "Pre" version
+
+## Problem Background
+
+Customers want to add the latest version of a package to their project, even tough it is a prerelease version.
+
+## Who are the customers
+
+All dotnet.exe customers
+
+## Goals
+
+Add an option to the command `add package` enable installing latest prerelease version.
+
+## Non-Goals
+
+## Solution
+
+Add the option `--prerelease` in the command `add package`. This command should get the latest version of the packet available even if it is not a stable version.
```

Comment:
You want to call out that this command will get the latest `listed` package. 

For reference see: https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/publish-nuget-package

Specifically https://docs.microsoft.com/en-us/nuget/nuget-org/policies/deleting-packages.

The idea behind listed/unlist packages is that, I as an author have decided that the package version I published had some issues and do not want it to be discoverable. So I unlist it. 

A user can explicitly request and use an unlisted version, but the author does not want it to be appear in search, like when you go to nuget.org and search: See example: https://www.nuget.org/packages?q=mypackage